src/core/Microsoft.PowerToys.Settings.UI/Views/GeneralPage.xaml.cs

```diff
@@ -131,16 +129,21 @@ private void Theme_Changed(object sender, RoutedEventArgs e)
         {
             RadioButton rb = sender as RadioButton;
 
-            if (rb != null)
-            {
-                string themeName = rb.Tag.ToString();
-                this.ReLoadTheme(themeName);
+            //if (rb != null)
```

Comment:
why comment this out?

Reply:
I will squash, merge and update the commented code. Thanks for pushing the PR in and for making the recommendation to move the UI to the pages themselves, it will definitely save us a lot of dev and test time!